test/python/quantum_info/test_sparse_observable.py

```diff
@@ -2008,3 +2008,71 @@ def test_term_pauli_base(self):
             Pauli("YYYII"),
         ]
         self.assertEqual([term.pauli_base() for term in obs], expected)
+
+    def test_to_sparse_list(self):
+        """Test converting to a sparse list."""
+        obs = SparseObservable.zero(100)
+        with self.subTest(msg="zero"):
+            self.assertEqual(0, len(obs.to_sparse_list()))
+
+        obs = SparseObservable.identity(100)
+        sparse_list = obs.to_sparse_list()
+        with self.subTest(msg="identity"):
+            self.assertEqual(1, len(sparse_list))
+            self.assertEqual("", sparse_list[0][0])
```

Comment:
Can we write these equality tests (etc) using the actual expected lists - that'll test length and full correctness in a single step, and give better error messages.

Reply:
Done in 358575b